scripts/Readme.md

```diff
@@ -0,0 +1,15 @@
+# 👁️‍🗨️ @monkvision/scripts
+
+To get all the Monitoring metrics
+
+``` yarn
+yarn run extract-metrics
+```
+
+We need a environment (.env) file at the same directory with below keys
+
+| **Keys**                 | **Description**                                                                              |
+|--------------------------|----------------------------------------------------------------------------------------------|
+| `TOKEN`                  | Sentry auth_token to access the api's                                                        |
+| `PROJECTS`               | List of all projects if user wants to filter else pass empty value (Ex. "")                  |
```

Comment:
Is this a comma-separated list ? It could be nice to specify in the doc I think

Reply:
Done